site/_posts/2019-07-02-Announcing-Hammer-8.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Announcing Hammer-8
+author: jprause
+date: 2019-07-02 08:35:39 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+We've just built Hammer-8. This release contains bug fixes, UI tweaks, and stabilization.
+Here are just a few of the things added since Hammer-7 release:
+
+- ADDED
+  * Add migration analytics reports
+
+- FIXED 
+  * Group AND expressions properly to account for nesting
+  * [V2V] Modify active_tasks so that it always reloads
+  * EvmDatabaseOps] Fix .validate_free_space target
```

Comment:
minor thing, but I think this is missing an opening square bracket `[`

Reply:
Good eyes. :eyes:

Reply:
It's a blessing and curse. Anyway thanks for the post, I will merge once the checks have completed!